en/cpp/README.md

```diff
@@ -51,6 +51,13 @@ The most important classes are:
 - [LogFiles](/api_reference/classdronecode__sdk_1_1_log_files.md): Download log files from the vehicle.
 
 
+The following APIs provide more direct access to underlying MAVLink messages/types.

```

Comment:
@julianoes Not in particular this bit - which separates out the direct APIs from the others. I don't know whether you wanted to say something else, but I figured worth explaining why there are multiple ways to access APIs.

Reply:
Ok!